index.php

```diff
@@ -1,17 +1,19 @@
 <?php
 
-require_once 'src/Watch.php';
 require_once 'src/WatchModel.php';
 require_once 'src/WatchViewHelper.php';
 
-
 $db = new PDO('mysql:host=db; dbname=watch_collection', 'root', 'password');
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
 $watchesModel = new WatchModel($db);
 $watches = $watchesModel->getAllWatches();
 
-
+if(isset($_POST['watch_id'])) {
+    $watch = new WatchModel($db);
+    $watch->restoreWatches($_POST['watch_id']);
+    $watches = $watch->getAllWatches(0);
```

Comment:
if you move this whole if statement to above line 10 (where you call getAllWatches the first time) you then won't need to call getAllWatches a second time.

Basically it means that you restore the watch before getting them out of the database.